sdk/attestation/Azure.Security.Attestation/src/AttestationAdministrationClient.cs

```diff
@@ -242,7 +246,15 @@ private async Task<AttestationResponse<string>> GetPolicyInternalAsync(Attestati
                 var token = AttestationToken.Deserialize(result.Value.Token);
                 if (_options.TokenOptions.ValidateToken)
                 {
-                    token.ValidateToken(_options.TokenOptions,GetSignersAsync(false, cancellationToken).EnsureCompleted(), cancellationToken);
+                    var signers = GetSignersAsync(false, cancellationToken).EnsureCompleted();
+                    if (!token.ValidateToken(_options.TokenOptions, signers, cancellationToken))
+                    {
+                        throw new AttestationTokenValidationFailedException("Attestation Token was rejected.")
```

Comment:
Consider creating either a constant for this string or a shared helper to construct the exception.  Should this string also indicate it was rejected by the customer's event with something like `$"Attestation Token was rejected by {nameof(AttestationTokenValidationOptions)}.{nameof(AttestationTokenValidationOptions.TokenValidated)} event handler."` to make is super clear the customer caused this themselves?